printer_mods/ZZP8202/Double_GEAR_ratio_Z/.metadata.yml

```diff
@@ -0,0 +1,34 @@
+title: Double_GEAR_ratio_Z
+description: "Double_GEAR_ratio_Z_belt_assembly_with_spherical_bearing_connection"
+mod_version: 1
+discord_username: ZZP8202
+cad:
+- 3D/Double_GEAR_ratio_Z_belt_assembly_with_Spherical_Bearing_connection.STEP
+stl:
+- STL/Adapter_x4.STL
+- STL/Pulley_Holder_x4.STL
+- STL/XY_Cable_Chain_Brdge_x1.STL
+- STL/Z_Bearing_Block_Bottom_With_Pulley_Left_A_x2.STL
+- STL/Z_Bearing_Block_Bottom_With_Pulley_Left_B_X2.STL
+- STL/Z_Bearing_Block_Bottom_With_Pulley_Right_A_x2.STL
+- STL/Z_Bearing_Block_Bottom_With_Pulley_Right_B_X2.STL
+- STL/Z_Bearing_Block_Top_X4.STL
+- STL/Z_Belt_Clamp_Bottom_x8.STL
+- STL/Z_Belt_Clamp_Upper_x8.STL
+images:
+- Images/CAD_Detal_view1.png
+- Images/CAD_Detal_view2.png
+- Images/CAD_front_view.png
+- Images/CAD_side_view.png
+- Images/Photo.jpg
+-Other/Assemble_Instructions/Pulley_Holder.gif
```

Comment:
There are some formatting errors in the YAML file. "The Other:" section should be first and then the subsequent entries need a space after the "-" in "-/Other/..."

Like so:
```
Other:
- Other/Assemble_Instructions/Pulley_Holder.gif
- Other/Assemble_Instructions/Z_Bearing_Block_Bottom_With_Pulley.gif
- Other/Assemble_Instructions/BOM/Pulley_Holder_X4_BOM.png
- Other/Assemble_Instructions/BOM/Z_Bearing_Block_Bottom_With_Pulley_BOM.png
- Other/Assemble_Instructions/BOM/Pulley_Holder_X4_BOM.pdf
- Other/Assemble_Instructions/BOM/Z_Bearing_Block_Bottom_With_Pulley_BOM.pdf

```